app/views/helpers/taxsInputTypeRadioGroup.scala.html

```diff
@@ -118,7 +118,7 @@
     }
 }
 
-<fieldset @fieldSetClass @fieldSetAttributes role="radiogroup" aria-required="true" @if(fieldErrors.nonEmpty) {aria-labelledby="@params.field.name-error-0"}>
+<fieldset @fieldSetClass @fieldSetAttributes @if(fieldErrors.nonEmpty) {aria-labelledby="@params.field.name-error-0"}>
```

Comment:
Why is this using aria-labelledby rather than aria-describedby?

Reply:
updated